zeebe/gateway-rest/src/main/java/io/camunda/zeebe/gateway/rest/config/ApiFiltersConfiguration.java

```diff
@@ -44,6 +50,19 @@ public FilterRegistrationBean<EndpointAccessErrorFilter> registerFilter(
     return registration;
   }
 
+  @ConditionalOnAuthenticationMethod(AuthenticationMethod.OIDC)
+  @Bean
+  public FilterRegistrationBean<EndpointAccessErrorFilter> registerUsersFilter(
+      final ObjectMapper objectMapper) {
+    final FilterRegistrationBean<EndpointAccessErrorFilter> registration =
+        new FilterRegistrationBean<>();
+    registration.setFilter(
+        new EndpointAccessErrorFilter(objectMapper, USERS_API_DISABLED_ERROR_MESSAGE));
+    registration.addUrlPatterns("/v2/users/*");
+    registration.setOrder(1);
+    return registration;
+  }
+
```

Comment:
It seems, my comment here is not shown. I've asked to rename the method to `disableUserApiFilter` to follow same pattern as `disableMultiTenancyApiFilter`. It also would be nice if you can rename the exisitng`registerFilter` method in this class.